bigquery/google/cloud/bigquery/routine.py

```diff
@@ -239,6 +240,17 @@ def body(self):
     def body(self, value):
         self._properties[self._PROPERTY_TO_API_FIELD["body"]] = value
 
+    @property
+    def description(self):
+        """Union[str, None]: Description of the routine (defaults to
```

Comment:
(nit)
Mypy [docs recommend](https://mypy.readthedocs.io/en/latest/kinds_of_types.html#optional-types-and-the-none-type) using the `Optional` shorthand instead of and `Union` with `None`.

```suggestion
        """Optional[str]: Description of the routine (defaults to
```

Reply:
ohk, should i change same in the other files which has description property like in  dataset.py and model.py .

Reply:
Strictly speaking it is not in scope of this PR, but if it's not too much work, it would be a welcome cleanup IMO. :+1: